docs/release-notes.rst

```diff
@@ -1,3 +1,216 @@
+                
+                "servicePort": 80,
+                
+                "shareNodes": true
+            }
+         ]
+
+Another possible workaround is to remove the erroneous applications and recreate them with different mappings so that each AS3 application will have a separate set of nodes.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/187
+
+------
+
+**BIG-IP reboots OR BIG-IP services restart if more than 60 endpoints are dynamically discovered in an APIC EPG**
+
+If an APIC endpoint group has more than 60 endpoints attached, then the endpoint list will not get reflected on the BIG-IP, and users may experience service restarts on BIG-IP. 
+
+**Workaround:** For any dynamic endpoint mapping, ensure that the number of endpoints in the corresponding APIC endpoint group never exceeds 60.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/188
+
+------
+
+**Node not removed from BIG-IP pool when node IP is a substring of some other node's IP**
+
+If a node (for example a node with IP 1.2.3.4) is deleted from APIC, and there is also another node 1.2.3.40 of which the original IP is a substring, it may be possible that the dynamic end point attach detach feature is not able to delete 1.2.3.4 from BIG-IP. Note: The pool members will get deleted as expected. 
+
+**Workaround:** Login to the BIG-IP UI and delete the problematic node
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/244
+
+------
+
+**Dynamic EP discovery does not work if duplicate IP pre-exists on a different partition.**
+
+If an APIC Tenant|App|EPG mapped to a BIG-IP pool has an endpoint with an IP address which already exists on the BIG-IP but in a different partition, then the APIC endpoint will not get added to BIG-IP pool. Also any successive configurations and endpoints also will not be discovered/deleted from this BIG-IP pool. 
+
+Workaround: Remove the duplicate IPs from the endpoint list on APIC and retry a manual sync of Endpoints from L4-L7 App Services --> Application Inventory --> Sync EPs icon. 
+
+Note: Similar issues might be seen with other erronous configurations such as unsupported IPv4 formats like 1.2.3.4/24 instead of 1.2.3.4
+
+AS3 Defect: https://github.com/F5Networks/f5-appsvcs-extension/issues/287
+
+------
+
+**Pool members deleted or added directly to BIG-IP don't get updated automatically after clicking "Sync EPs".**
+
+1. If BIG-IP pool members automatically get added by **Dynamic endpoint discovery** feature, but then few endpoints are deleted directly from the BIG-IP (i.e. out of band); these endpoints do not get created again on clicking **L4-L7 App Services --> Application Inventory --> Sync EPs** for that application. 
+
+2. Similarly, when few pool members are added directly to the BIG-IP (i.e. out of band), these extra members are not deleted after clicking **L4-L7 App Services --> Application Inventory --> Sync EPs**
```

Comment:
not sure what 'few pool members' means here, does this mean "when a few pool members.." ie more than a couple, but some arbitrary number?  I'd suggest either "..when a few pool members", or maybe better "when pool members are added..."